errdefs/defs.go

```diff
@@ -52,3 +57,18 @@ type ErrNotImplemented interface {
 type ErrUnknown interface {
 	Unknown()
 }
+
+// ErrCancelled signals that the action was cancelled.
+type ErrCancelled interface {
+	Cancelled()
+}
+
+// ErrDeadline signals that the deadline was reached before the action completed.
+type ErrDeadline interface {
+	DeadlineExceeded()
+}
+
+// ErrDataLoss indicates that data was lost or there is data corruption.
+type ErrDataLoss interface {
```

Comment:
Looks like `ErrDataLoss` is not used anywhere


I see some mention of these but for `gRPC` errors; https://github.com/moby/moby/blob/6540d5581473c292ade46ee1b620172441943f4b/api/server/httputils/errors.go#L122-L128

Reply:
I plan to :)
e.g. for corrupted json log entries.